ui/app/components/app/connected-accounts-permissions/connected-accounts-permissions.component.js

```diff
@@ -57,7 +58,8 @@ export default class ConnectedAccountsPermissions extends PureComponent {
           <ul className="connected-accounts-permissions__list">
             {permissions.map(({ key: permissionName }) => (
               <li key={permissionName} className="connected-accounts-permissions__list-item">
-                <i className="fas fa-check-square" />{t(permissionName)}
+                <CheckBox checked disabled id={permissionName} className="connected-accounts-permissions__checkbox" />
```

Comment:
For consistency with the approval component, and future-proofing, we should probably compute an `isDisabled` variable that's `true` if `permissionName === 'eth_accounts'`.

Reply:
I think it would make sense to create a `PermissionCheckbox` component that combines all of this functionality for reuse. `CheckBox` does not include a label by default for example. Whereas a label is required for a permissions based checkbox. I'll also be cleaner than duplicating the logic in two places and will be reusable if these permissions ever appear in another location.

Reply:
Yeah, good point. I'm fine with this as-is, or if you introduce that component here.

Reply:
Ultimately the permission controller should probably determine which permissions are required or not, if we're still planning on letting dapps specify whether requested permissions are optional or required. So we might want to migrate this `eth_accounts` exception into the permissions controller for now instead of into a shared UI component.

Reply:
I took a stab at doing this work locally, even in the event that we move some logic around I do think it makes sense to have the list of permissions rendered in one place. I'll keep the branch alive and return to it when it makes sense -- there are some gotchas that would overly complicate this work which began as simply changing icon color.